PMM-14630: Use correct casing in navigation menu and Help Center

dmitri-saricev-3pillargloball opened the pull request:
https://perconadev.atlassian.net/browse/PMM-14630

results:
Link to the Feature Build:
<img width="2154" height="1088" alt="Screenshot 2025-12-12 at 16 01 43" src="https://github.com/user-attachments/assets/e03ab16e-0811-40e8-881b-be4df51dbdf3" />
<img width="1175" height="1268" alt="Screenshot 2025-12-12 at 16 04 34" src="https://github.com/user-attachments/assets/f8fe44f0-9e35-490a-8f90-e44d38d9f457" />
<img width="1987" height="1239" alt="Screenshot 2025-12-12 at 16 04 48" src="https://github.com/user-attachments/assets/942e2cd5-96cd-4070-a2a1-bf660a2cc4d0" />
<img width="649" height="1140" alt="Screenshot 2025-12-12 at 16 05 49" src="https://github.com/user-attachments/assets/924acc50-961d-437d-893e-8a66fbed4ebb" />
<img width="484" height="1348" alt="Screenshot 2025-12-12 at 16 06 11" src="https://github.com/user-attachments/assets/ca424ab6-4df6-4120-9035-c72f5b49fe43" />
<img width="820" height="890" alt="Screenshot 2025-12-12 at 16 07 07" src="https://github.com/user-attachments/assets/00ca54d7-33e6-43d6-a75d-ec7acd8b5ac5" />

FB
https://github.com/Percona-Lab/pmm-submodules/pull/4152
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm3-submodules/detail/pmm3-submodules/2067/pipeline/



If this PR adds, removes or alters one or more API endpoints, please review and add or update the relevant [API documentation](https://github.com/percona/pmm/tree/v3/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

dmitri-saricev-3pillargloball committed:
PMM-14630: Use correct casing in navigation menu and Help Center

matejkubinec committed:
Merge branch 'v3' into PMM-14630-use-correct-casing-in-navigation-menu-and-help-center

dmitri-saricev-3pillargloball committed:
Merge branch 'v3' into PMM-14630-use-correct-casing-in-navigation-menu-and-help-center

dmitri-saricev-3pillargloball committed:
PMM-14630: Improve text casing and date formatting in Updates UI

- Change button text: 'Check Updates Now' to 'Check updates now'
- Change button text: 'PMM Home' to 'PMM home'
- Update footer date format to 'Month Day, Year, HH:MM UTC'
- Remove duplicate 'Last checked' text from update card
- Improve consistency across Updates page components

dmitri-saricev-3pillargloball committed:
Merge branch 'v3' into PMM-14630-use-correct-casing-in-navigation-menu-and-help-center

matejkubinec commented on ui/apps/pmm/src/pages/updates/update-card/UpdateCard.tsx:
```diff
@@ -118,14 +117,6 @@ export const UpdateCard: FC = () => {
               {data?.latest && formatVersion(data.latest)}
             </Typography>
           )}
-          {data.lastCheck && (
```
@dmitri-saricev-3pillargloball why was this removed?

dmitri-saricev-3pillargloball replied:
Design changed you can check screenshot in task.

matejkubinec replied:
I don't see it being removed from the card just the footer content be synced with the update card.

dmitri-saricev-3pillargloball replied:
Matej I am not understanding we can discuss this in slack and this changes was approved by Catalina.

dmitri-saricev-3pillargloball replied:
Matej I am not understanding we can discuss this in slack and this changes was approved by Catalina.

dmitri-saricev-3pillargloball committed:
PMM-14630: Fix text casing in navigation menu and tour messages

1. navigation.constants.ts - fixed 3 items:
   - 'CPU Utilization' → 'CPU utilization'
   - 'Percona Advisors' → 'Advisors'
   - 'Advisor Insights' → 'Advisor insights'

2. navigation.utils.ts - fixed 1 item:
   - '${capitalize(category)} Advisors' → '${capitalize(category)} advisors'
   - Now dynamically generates submenu items: "Security advisors", "Configuration advisors", etc.

3. alerting.messages.ts - all titles converted to sentence case:
   - 'Fired Alerts' → 'Fired alerts'
   - 'Alert Templates' → 'Alert templates'
   - And others

4. product.messages.ts:
   - 'Percona Advisors' → 'Advisors'

dmitri-saricev-3pillargloball committed:
Merge branch 'v3' into PMM-14630-use-correct-casing-in-navigation-menu-and-help-center

dmitri-saricev-3pillargloball committed:
PMM-14630: Revert "Checked on:" to "Last checked:"

Based on feedback from Davi and Catalina, both instances should
use "Last checked:" consistently throughout the application.

dmitri-saricev-3pillargloball committed:
Merge branch 'v3' into PMM-14630-use-correct-casing-in-navigation-menu-and-help-center
